jenkins/performance-search5-lucene.groovy

```diff
@@ -0,0 +1,52 @@
+pipeline {
+    agent any
```

Comment:
We have separate nodes for performance tests, please use those.

```suggestion
    agent 'Performance'
```

Reply:
If I try to apply the change, I got the following error: https://ci.hibernate.org/job/hibernate-search-performance/job/search5-lucene/10/console.

I'm finding another way to chose the node.

Reply:
The following seems to be working:

``` groovy
agent {
  label 'Performance'
}
```